nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/json/JsonPathReader.java

```diff
@@ -48,15 +48,17 @@
 import com.jayway.jsonpath.JsonPath;
 
 @Tags({"json", "jsonpath", "record", "reader", "parser"})
-@CapabilityDescription("Parses JSON records and evaluates user-defined JSON Path's against each JSON object. The root element may be either "
-    + "a single JSON object or a JSON array. If a JSON array is found, each JSON object within that array is treated as a separate record. "
-    + "User-defined properties define the fields that should be extracted from the JSON in order to form the fields of a Record. Any JSON field "
-    + "that is not extracted via a JSONPath will not be returned in the JSON Records.")
+@CapabilityDescription("Parses JSON records and evaluates user-defined JSON Path's against each JSON object. The reader does not require the "
```

Comment:
I would be hesitant to indicate "The reader does not require the flow file content to be well-formed JSON." This gives me the impression that improper JSON will still be handled correctly, perhaps by skipping over the invalid parts? Perhaps we should word it as "While the reader expects each record to be well-formed JSON, the content of a FlowFile may consist of many records, either as a well-formed JSON array, or a series of JSON records with optional whitespace between them, such as the common 'JSON-per-line' format." or something of that nature.